colossalai/ppinference/microbatch_manager.py

```diff
@@ -0,0 +1,144 @@
+from typing import Dict
+
+import torch
+
+from .inference_config import InferenceConfig
+
+__all__ = 'MicroBatchManager'
+
+PREFILL = 1
+GENERATE = 2
+DONE = 3
+
+
+class MicroBatchDescription():
+
+    def __init__(
+        self,
+        mb_inputs: torch.Tensor,
+        interval_inputs,
+        new_length: int,
+    ) -> None:
+        if mb_inputs is not None:
+            assert mb_inputs.get('input_ids') is not None and mb_inputs.get('attention_mask') is not None
```

Comment:
It seems that type hint of `mb_inputs` is wrong.

Reply:
done